job-tprocc-mariadb-ephemeral.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-tprocc-mariadb-ephemeral
+spec:
+  template:
+    spec:
+      initContainers:
+        - image: docker.io/robertbartlettbaron/k-hammerdb:latest
```

Comment:
do we have a NERC docker.io image location we should be using?

Reply:
Do we?  We if we don't, we probably should.

Reply:
We don't currently but I suppose we could create one if folks prefer. We've used github repo registries for this purpose given the integration with github actions. For example:

https://github.com/nerc-project/coldfront-nerc/pkgs/container/coldfront-nerc
https://github.com/nerc-project/coldfront-nerc/blob/main/.github/workflows/docker.yaml

Reply:
@rob-baron will this work for your purposes?

Reply:
I will work on placing this in a GitHub repo.